ui/help.html

```diff
@@ -601,7 +601,7 @@
   </div>
   <div class="col-xs-6">
     <a class="midgray pull-right"
-       href="../media/revision">Build</a>
+       href="http://mzl.la/1gQ8xCi">WhatsDeployed</a>
```

Comment:
I would use the full link rather than an abbreviation here. This isn't twitter, I like to know where a hyperlink will take me. :)